transit_odp/publish/templates/publish/snippets/disruptions_map.html

```diff
@@ -0,0 +1,55 @@
+<head>
+  <style>
```

Comment:
Can this be moved to css file instead in inline

Reply:
done